cmd/skaffold/app/cmd/flags.go

```diff
@@ -182,6 +182,15 @@ var flagRegistry = []Flag{
 		DefinedOn:     []string{"dev", "build", "run", "debug", "deploy", "render", "apply", "test"},
 		IsEnum:        true,
 	},
+	{
+		Name:          "wait-for-connection",
+		Usage:         "Blocks execution until the /v2/events gRPC/HTTP endpoint is hit",
+		Value:         &opts.WaitForConnection,
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"dev", "debug"},
```

Comment:
Should the commands this is `DefinedOn` here be the same as those for `--enable-rpc`?  Might make sense as any command can initialize the gRPC/HTTP server and emit events

Reply:
yeah, this was definitely a brain fart of mine 😅 thanks for catching